lib/osf-components/addon/components/search-help-modal/template.hbs

```diff
@@ -1,7 +1,7 @@
 <div id='ember-bootstrap-modal-container'>
     <OsfDialog
-        @isOpen={{this.isOpen}}
-        @onClose={{action (mut this.isOpen) false}}
+        @isOpen={{@isOpen}}
+        @onClose={{action (mut @isOpen) false}}
```

Comment:
This doesn't look right. You don't change args, you change properties on the component class. This looks like it should be reverted. See above for further suggestions.